src/meshes/t8code_mesh.jl

```diff
@@ -513,7 +519,7 @@ function adapt_callback_wrapper(forest,
 end
 
 """
-    Trixi.adapt!(mesh::T8codeMesh, adapt_callback; kwargs...)
+    adapt!(mesh::T8codeMesh, adapt_callback; kwargs...)
```

Comment:
```suggestion
    Trixi.adapt!(mesh::T8codeMesh, adapt_callback; kwargs...)
```
`adapt!` is not exported, right?